libs/geospatial/src/test/java/org/codice/ddf/libs/location/CountryCodeConverterTest.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.libs.location;
+
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+import java.util.List;
+import java.util.MissingResourceException;
+
+import org.junit.Test;
+
+public class CountryCodeConverterTest {
+
+    @Test
+    public void testFipstoIso3Conversion() {
+        String iso3 = CountryCodeConverter.convertFipsToIso3("RS");
+        assertThat(iso3, is("RUS"));
+    }
+
+    @Test
+    public void testFipsToIso3ConversionInvalidCountryCode() {
+        String iso3 = CountryCodeConverter.convertFipsToIso3("");
+        assertThat(iso3, nullValue());
+    }
+
+    @Test
+    public void testIso3ToFipsConversion() {
+        List<String> fipsCountryCodes = CountryCodeConverter.convertIso3ToFips("RUS");
+        assertThat(fipsCountryCodes.get(0), is("RS"));
```

Comment:
We should use `containsInAnyOrder("RS")` here to make sure that only one country code was returned.